lib/SymbolGraphGen/DeclarationFragmentPrinter.cpp

```diff
@@ -189,9 +189,17 @@ void DeclarationFragmentPrinter::printAbridgedType(const GenericTypeDecl *TD,
       case DeclKind::Protocol:
         printText(getTokenText(tok::kw_protocol));
         break;
-      case DeclKind::Class:
-        printText(getTokenText(tok::kw_class));
-        break;
+        case DeclKind::Class: {
```

Comment:
The indent is different of the above statements. Is is on purpose or a mistake?

Reply:
Heh - Xcode doesn't always maintain formatting well. This should be fixed in an update shortly.

Reply:
Reopen the conversation as the indent still looks wrong to me.

Reply:
Huh - thanks @Kyle-Ye, I'll look closer at tabs vs spaces here, I thought I'd fixed it, but I'm new to contributing here.

Reply:
If you are using Xcode as your IDE, you may change the setting(Tab Width and Indent Width) here to match the existing code.

<img width="942" alt="image" src="https://github.com/apple/swift/assets/43724855/7442db7a-3184-4f46-b8b9-d9dd93df348a">